launch/duro_example.launch

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <launch>
-  <param name="duro_address" value="192.168.1.10"/>
-  <param name="duro_port" value="55555"/>
-  <node pkg="duro-ros" type="duronode" name="gps_duro_1" />
-</launch>
\ No newline at end of file
+    <group ns="gps/duro">
+        <node pkg="duro_ros" type="duronode" name="duro_driver">
+            <param name="duro_address" value="192.168.1.10"/>
+            <param name="duro_port" value="55555"/>
+            <param name="gps_receiver_frame_id" value="duro_link"/>
+            <param name="imu_frame_id" value="duro_link"/>
+            <param name="utm_frame_id" value="utm"/>
```

Comment:
I think it is better to use `map` `utm_frame_id`  as an example, but this works too. The others seems ok. And the  `utm_frame_id`  is also acceptable this way.